components/organisms/messaging/components/fullPreview/index.js

```diff
@@ -0,0 +1,271 @@
+import { useEffect, useRef, useState } from 'react'
+
+import styled, { css } from 'styled-components'
+
+import { useComponentCommunication, Preview } from '../../../../'
+import { MessageNames, MessagingActions, MessagingSubscriber } from '../../../../services'
+import { Arrow } from './arrow'
+import { Cross } from './cross'
+import { DownloadButton } from './download'
+
+export const FullPreview = () => {
+  const [selectedFileIndex, setselectedFileIndex] = useState(null)
```

Comment:
the correct naming for these kind of states are [selectedFileIndex, setSelectedFileIndex]. Please correct this in other states as well